specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json

```diff
@@ -4928,6 +5449,267 @@
+          "description": "Specifies the unique of the dedicated physical machine on which the dedicated host resides."
+        },
+        "capacity": {
+          "$ref": "#/definitions/DedicatedHostCapacity",
+          "description": "The total capacity of the dedicated host."
+        },
+        "availableCapacity": {
+          "$ref": "#/definitions/DedicatedHostAvailableCapacity",
+          "description": "Unutilized capacity of the dedicated host."
+        },
+        "statuses": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/InstanceViewStatus"
+          },
+          "description": "The resource status information."
+        }
+      },
+      "description": "The instance view of a dedicated host."
+    },
+    "DedicatedHostProperties": {
+      "properties": {
+        "platformFaultDomain": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Fault domain of the host within a group. Supported values 0,1,2."
+        },
+        "autoReplaceOnFailure": {
+          "type": "boolean",
+          "description": "Whether the host should be replaced automatically in case of a failure. The value is defaulted to true when not provided."
+        },
+        "hostId": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique id generated and assigned to the dedicated host by the platform."
+        },
+        "virtualMachines": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SubResourceReadOnly"
+          },
+          "readOnly": true,
+          "description": "A list of references to all virtual machines in the Dedicated Host."
+        },
+        "licenseType": {
+          "$ref": "#/definitions/DedicatedHostLicenseType",
+          "description": "Specifies the software license type that will be applied to the VMs deployed on the dedicated host. <br><br> Possible values are: <br><br> **None** <br><br> **Windows_Server_Hybrid** <br><br> **Windows_Server_Perpetual** <br><br> Default: **None**"
+        },
+        "provisioningTime": {
+          "readOnly": true,
+          "type": "string",
+          "format": "date-time",
+          "description": "The date when the host was first created."
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The provisioning state, which only appears in the response."
+        },
+        "instanceView": {
+          "$ref": "#/definitions/DedicatedHostInstanceView",
+          "readOnly": true,
+          "description": "The dedicated host instance view."
+        }
+      },
+      "description": "Properties of the dedicated host."
+    },
+    "DedicatedHost": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DedicatedHostProperties"
+        },
+        "sku": {
```

Comment:
Should this be sku or something else?

Also this property / its definition was not defined